src/imaging.py

```diff
@@ -63,36 +68,49 @@
 GAIN = args.gain
 IMG_WIDTH = args.image_width
 IMG_HEIGHT = args.image_height
+SUN_ALT = args.sun_alt
+SUN_AZ = args.sun_az
 
+# Calibration and position determination
 antAlt, antAz = calibrate(ANT_OFFSET_EL, ANT_OFFSET_AZ)
 sunAlt, sunAz = getSunPosition(LAT, LON)
 
-startingAlt = sunAlt - (IMG_HEIGHT / 2)
-finalAlt = sunAlt + (IMG_HEIGHT / 2)
-startingAz = sunAz - (IMG_WIDTH / 2)
-finalAz = sunAlt + (IMG_WIDTH / 2)
+# If sun altitude and azimuth is specified, use that instead (testing)
+if (SUN_ALT != 400): sunAlt = SUN_ALT
+if (SUN_AZ != 400): sunAz = SUN_AZ
 
+# Calculate the starting Altitude and Azimuth based on the sun's position and image dimensions
+startingAlt = int(sunAlt - (IMG_HEIGHT / 2))
+startingAz = int(sunAz - (IMG_WIDTH / 2))
```

Comment:
Why do startingAlt and startingAz have to be integers?  They're only used in getDifferenceDeg and they can be non-integers.  I think we were only getting issues before with finalAlt and finalAz because they were being used as the range of the for loop.  Thoughts? @jessica-guetre @bng919

Reply:
Yeah I don't think the starting point needs to be int, as long as we are stepping by 1 degree step sizes and finalAlt/finalAz are ints like you said.

Reply:
Good point
Just pushed a removal of the int casting